### ISSUE TITLE
Move does not work when the window is moved offscreen and then resized

### ISSUE THREAD

**Issue by philc** [jigish/slate#41]
This move command moves the window, but does not properly resize it. This is as of version f3f3982.

On screen 1, hit CTRL+CMD+left. This will resize the window to fill the left half of the screen. Next, hit CTRL+CMD+SHIFT+down. This should move and resize the window to fill the bottom left corner of the screen, but the window remains its original size.

```
bind left:ctrl;cmd push left bar-resize:screenSizeX/2
bind down:ctrl;shift;cmd move screenOriginX;screenOriginY+screenSizeY/2 screenSizeX/2;screenSizeY/2
```

Note that this move command _does_ properly move and resize the window:

```
bind up:ctrl;shift;cmd move screenOriginX+screenSizeX/2;screenOriginY screenSizeX/2;screenSizeY/2
```

Note that this is with a monitor configuration of

```
1   3
  2
```

**Comment by jigish** [comment]
Yah I noticed this as well and opened Issue #34. It is fixed, I just haven't generated another release yet as I'm trying to fix the other bugs in window hints as well as create the config helper in the next release.

**Comment by jigish** [comment]
going to close this issue. maybe i'll generate a release tonight and worry about the other bugs in the one after that.

**Comment by philc** [comment]
Dang it. You are too fast for me.

**Comment by jigish** [comment]
this has been fixed better in 0.4.18. The fix I did worked for the most part, but there were still minor issues.
